src/views/shared/FooterSignout.js

```diff
@@ -17,31 +17,32 @@ const { Util } = internal.util;
 export default View.extend({
   template: hbs(
     '\
-      <a href="#" class="link {{linkClassName}}" data-se="signout-link">\
+      <a href="#" class="link signout {{linkClassName}}" data-se="signout-link">\
```

Comment:
why want to add `signout` all the time. It was either `signout` or `back` before?

Reply:
I added class `signout` to bind click event `handleSignout` only for this link and to not bind it to custom help link.
I've removed class `signout` and now use `click a[data-se="signout-link"]` as selector for click event that will b handles with `handleSignout`